packages/utils/telemetry-utils/src/test/config.spec.ts

```diff
@@ -22,7 +22,8 @@ const getMockStore = (settings: Record<string, string>): Storage => {
 	return {
 		getItem: (key: string): string | null => {
 			ops.push(key);
-			return settings[key];
+			// eslint-disable-next-line unicorn/no-null
+			return settings[key] ?? null;
```

Comment:
What's the rationale here? Is the calling code prepared to handle `null` instead of `undefined`?

Reply:
yes, getItem is typed to string | null, but returning `settings[key]` results in a type of string | undefined, so I just force it to null if its undefined